components/AdCard.vue

```diff
@@ -1,7 +1,9 @@
 <template>
-  <v-alert border="bottom">
-    I'm a card of type ad with a link {{ card.link }}.
-  </v-alert>
+  <div class="card">
+    <v-alert border="bottom">
+      I'm a card of type ad with a link {{ card.link }}.
+    </v-alert>
```

Comment:
I think we can remove Vuetify.
I will work on the real html/css for the cards in #8 and #9

Reply:
Done 👍